### PR TITLE
Styling

### PR DESCRIPTION
**Pull request by arabiu033**
## Changes made in this PR :bookmark_tabs: 

- Refactor some old code to meet standards
- Add React CSS styling for each component
- Try to match the given design
- Fix all linters errors

### CODE REVIEW

File: src/css/book.module.css [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+  margin: 0.188rem 20.188rem 0 0;
+  font-family: "Roboto Slab", serif;
+  font-size: 1.5rem;
+  letter-spacing: -0.2px;
+  color: var(--black-two);
+  font-weight: bold;
+}
+
+.author {
+  /* margin: 0.25rem 0 0; */
```
Comment (mahbubul14) [inline src/css/book.module.css]: - [x] Kindly remove the inline comments from your code. The old pieces of code as inline comments make your project look messy. If you need to review a previous version of your code, you can always use git history. :+1:

**Review by mahbubul14 — CHANGES_REQUESTED** [review]
Hi @arabiu033,

Good job so far! :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!

# Required Changes ♻️

- [ ] Kindly check the inline comments under the review. :arrow_down: 

###  To Highlights
:heavy_check_mark: No linter error.
:heavy_check_mark: Proper Github flow used.
:heavy_check_mark: Descriptive PR title & description.

### Optional suggestions

Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better.

- [ ] In the stylesheets of the app, kindly make sure that basic CSS bad practices are not used [(article for support)](https://speckyboy.com/good-bad-css-practices/) like no repeating of the same rule.

Note: One of the following examples is: You have used `background-color: #f5f6fa;` several times.


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. You can also reach me on [slack](https://microverse-students.slack.com/team/U01J39GL614)

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: src/css/book.module.css (hunk 1 above)
Comment (arabiu033) [reply src/css/book.module.css]: @mahbubul14 Could you please elaborate on the CSS blunders? I am aware of the rules and have tried to abide by them. However, I'm perplexed that you still believe I'm not:

Could you please take some more screenshots for me?

Comment (arabiu033) [reply src/css/book.module.css]: I've seen some and fixed them. Please can you check again?

File: src/css/addBook.module.css [hunk 2]
```diff
@@ -0,0 +1,52 @@
+.container {
+  width: 100%;
+  display: flex;
```
Comment (GzimAsani) [inline src/css/addBook.module.css:3]: - [ ] As the previous reviewer mentioned you should not use  CSS bad practices[(article for support)](https://speckyboy.com/good-bad-css-practices/) like no repeating of the same rule. For example you have used `display: flex` 7 times. I would suggest you create a class example: **d-flex** and use it at all places that you need on your code

**Review by GzimAsani — CHANGES_REQUESTED** [review]
### Hello  @arabiu033            :wave:   
 Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there!
 
 ## Highlights
 * Good job with the removing the commented code :+1:  :+1: 

 ## Required Changes 

- [ ] Check the comments below :arrow_double_down: 







 ## Optional suggestions
 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding! clapclapclap
 
 Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
 
 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/css/addBook.module.css (hunk 2 above)
Comment (arabiu033) [reply src/css/addBook.module.css:3]: Lol :laughing: I can't believe I'm getting lessons on CSS again. Thank you, man :+1: Let me implement that.

**Comment by arabiu033** [comment]
Please pass this on to the next reviewer. I'd like to point out that I'm not using standard CSS files. In an effort to learn something new every day, I attempted to style the components in the React CSS module styling. So I have a separate 'file.module.css' for each component.

If I had simply used the standard styling method, I would not have encountered this problem. But I did make some changes and removed all the redundancy I could without breaking the styling.

**Review by elisha2kyakpo1 — APPROVED** [review]
Hi 👋 @arabiu033, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- You did well to implement the requested changes✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**